ImageKitSample/Program.cs

```diff
@@ -0,0 +1,426 @@
+
+            // CopyFolder
+            CopyFolderRequest cpyFolderRequest = new CopyFolderRequest
+            {
+                SourceFolderPath = "path_1",
+                DestinationPath = "path_2",
+                IncludeFileVersions = true,
+            };
+
+            ResultOfFolderActions resultOfFolderActions = imagekit.CopyFolder(cpyFolderRequest);
+
+            // MoveFolder
+            MoveFolderRequest moveFolderRequest = new MoveFolderRequest
+            {
+                SourceFolderPath = "path_1",
+                DestinationPath = "path_2",
+            };
+
+            ResultOfFolderActions resultOfFolderActions1 = imagekit.MoveFolder(moveFolderRequest);
+
+            #endregion
+
+            #region GetBulkJobStatus
+
+            ResultBulkJobStatus resultBulkJobStatus = imagekit.GetBulkJobStatus("job_Id");
+
+            #endregion
+
+            #region  Purge
+
+            ResultCache resultCache = imagekit.PurgeCache("url");
+
+            ResultCacheStatus resultCacheStatus = imagekit.PurgeStatus("request_Id");
+
+            #endregion
+
+            #region  Metadata
+            ResultMetaData resultMetaData = imagekit.GetFileMetadata("file_Id");
+
+            ResultMetaData resultMetaData1 = imagekit.GetRemoteFileMetadata("https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg");
+
+            // CustomMetaDataFields
+            ResultCustomMetaDataFieldList resultCustomMetaDataFieldList = imagekit.GetCustomMetaDataFields(true);
+
+            // CreateCustomMetaDataFields
+            CustomMetaDataFieldCreateRequest requestModel = new CustomMetaDataFieldCreateRequest
+            {
+                Name = "field_name",
+                Label = "field_lable",
+            };
+            CustomMetaDataFieldSchemaObject schema = new CustomMetaDataFieldSchemaObject
+            {
+                Type = CustomMetaDataFieldSchemaObject.CustomMetaDataTypeEnum.Number,
+                MinValue = 1000,
+                MaxValue = 3000,
+                MinLength = 500,
+                MaxLength = 600,
+                IsValueRequired = false,
+            };
+
+            requestModel.Schema = schema;
+            ResultCustomMetaDataField resultCustomMetaDataField = imagekit.CreateCustomMetaDataFields(requestModel);
+
+            // UpdateCustomMetaDataFields
+            CustomMetaDataFieldUpdateRequest requestUpdateModel = new CustomMetaDataFieldUpdateRequest
+            {
+                Id = "field_id",
+            };
+            CustomMetaDataFieldSchemaObject updateschema = new CustomMetaDataFieldSchemaObject
+            {
+                Type = CustomMetaDataFieldSchemaObject.CustomMetaDataTypeEnum.Number,
+                MinValue = 1000,
+                MaxValue = 3000,
+                MinLength = 500,
+                MaxLength = 600,
+                IsValueRequired = false,
+            };
+
+            requestUpdateModel.Schema = schema;
```

Comment:
It should have been `requestUpdateModel.Schema = updateschema;`
It is not tested.